### PR TITLE
[SWARM-1978] Add the Swarm Cache Booster

### PR DESCRIPTION
**Pull request by sberyozkin**

### CODE REVIEW

**Comment by gastaldi** [comment]
[test]

**Comment by gastaldi** [comment]
[test]

**Comment by Ladicek** [comment]
@sberyozkin Do you intend to add the `-redhat` variant to this PR, or later PR? I think we should add it to this PR, because it depends on #190 and can't be merged sooner anyway.

File: wildfly-swarm/community/cache/booster.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+metadata:
+  runsOn: 
+    - '!starter'
+    - '!osio'
+name: WildFly Swarm - Cache
+description: Demonstrate how to use a cache server in Wildfly Swarm
```
Comment (Ladicek) [inline wildfly-swarm/community/cache/booster.yaml]: "Wildfly" -> "WildFly"

**Comment by gastaldi** [comment]
#190 is merged, feel free to merge this PR anytime

**Comment by gastaldi** [comment]
[test]

**Comment by gastaldi** [comment]
[test]

**Comment by gastaldi** [comment]
[test]

**Pull request merged**